js/src/wp-seo-admin-global.js

```diff
@@ -128,3 +128,55 @@
 	window.wpseoMakeDismissible = wpseoMakeDismissible;
 	window.wpseoDismissLink = wpseoDismissLink;
 }());
+
+(function() {
```

Comment:
See .. https://github.com/Yoast/wordpress-seo/pull/4327/commits/8fa7e87f85d09e8c15b68b12f205ef7b2cf1c97a
Doesn't really matter that much I guess, prefer suggested method as commit shows.